priv/gettext/pt/LC_MESSAGES/book.po

```diff
@@ -45,70 +43,71 @@ msgstr ""
 #, elixir-format
 #: lib/listudy_web/templates/book/recommended.html.eex:1
 msgid "Chess Books"
-msgstr ""
-
+msgstr "Livros de xadrez"
 #, elixir-format
 #: lib/listudy_web/templates/book/recommended.html.eex:1
 msgid "Grandmaster recommended"
-msgstr ""
-
+msgstr "Recomendado por Mestres"
 #, elixir-format
 #: lib/listudy_web/templates/book/recommended.html.eex:10
 msgid "Have you found a recommendation from a grandmaster that is not yet in the list? Send me an email and I will add it."
 msgstr ""
-
+"Você encontrou uma recomendação de um grande mestre que ainda não está na "
+"lista? Envie-me um e-mail e eu o adicionarei."
```

Comment:
The syntax of gettext is quite strict on how the translations have to be done. The whole translation has to be in one string quoted by "", and it has to be after the `msgstr`. Can you please fix where this happens in your translations.

Reply:
Gotcha ok will do started by rebasing it will revisit tonight thanks